common/components/BalanceSidebar/EquivalentValues.tsx

```diff
@@ -63,23 +68,19 @@ export default class EquivalentValues extends React.Component<Props, CmpState> {
               {key}:
             </span>{' '}
             <span className="EquivalentValues-values-currency-value">
-              {balance.isPending ? (
-                <Spinner />
-              ) : (
-                <UnitDisplay
-                  unit={'ether'}
-                  value={balance ? balance.muln(rates[currency][key]) : null}
-                  displayShortBalance={2}
-                />
-              )}
+              <UnitDisplay
```

Comment:
Why get rid of the spinner?

Reply:
The spinner should never happen; It was replaced with the one large spinner on line 83.

Reply:
Un-redacted, this is fixed.